sw/device/silicon_creator/lib/sigverify/spx_verify.c

```diff
@@ -62,20 +62,69 @@ static const uint32_t kSpxVerifyShares[kSigverifySpxRootNumWords] = {
     0x9eabb3c3,
 };
 
+/**
+ * Domain-separation prefix for SPHINCS+ with no prehashing.
+ *
+ * The domain separation prefix is the following byte sequence:
+ *   0x00 || len(ctx) || ctx
+ *
+ * In our case, `ctx` is always the empty string, so the length is 0.
+ */
+static const uint8_t kSpxVerifyPureDomainSep[] = {
+    0x00,
+    0x00,
+};
+
+/**
+ * Domain-separation prefix for SPHINCS+ with SHA256 prehashing.
+ *
+ * The domain separation prefix is the following byte sequence:
+ *   0x01 || len(ctx) || ctx || OID(PH)
+ *
+ * In our case, `ctx` is always the empty string and PH (the pre-hashing
+ * function) is always SHA256.
+ */
+static const uint8_t kSpxVerifyPrehashDomainSep[] = {
```

Comment:
I think this is the right byte ordering. Also the right OID identifier for `SHA-2-256`.

```
OID ← toByte(0x0609608648016503040201, 11) ▷ 2.16.840.1.101.3.4.2.1
```